scripts/utils.ts

```diff
@@ -0,0 +1,19 @@
+import fs from 'node:fs';
+
+export const parseJsonFile = <T>(filePath: string, raiseError = true) => {
+  try {
+    if (fs.existsSync(filePath)) {
+      return JSON.parse(fs.readFileSync(filePath, { encoding: 'utf-8' })) as T;
+    }
+    if (raiseError) {
+      throw new Error(`The file ${filePath} doesn't exist`);
+    }
```

Comment:
Honestly, I don't like it at all, I just kept this "safe" behaviour because I wanted to use the same code for the `getExistingConfig` in the deploy script. I'll review it again to understand if I can get rid of it.